components/nexusoperations/executors.go

```diff
@@ -339,7 +339,11 @@ func (e taskExecutor) saveResult(ctx context.Context, env hsm.Environment, ref h
 					case string((&commonpb.Link_WorkflowEvent{}).ProtoReflect().Descriptor().FullName()):
 						link, err := ConvertNexusLinkToLinkWorkflowEvent(nexusLink)
 						if err != nil {
-							// silently ignore for now
+							// TODO(rodrigozhou): ignore error for now
```

Comment:
This TODO isn't clear, we're already ignoring the error.

Reply:
Add better description.